content-src/asrouter/templates/OnboardingMessage/ToolbarBadgeMessage.schema.json

```diff
@@ -21,6 +21,16 @@
     "delay": {
       "type": "number",
       "description": "Optional delay in ms after which to show the notification"
+    },
+    "aria-label": {
```

Comment:
I'd rather we didn't use the name aria-label here. While aria-label is about a11y labelling, aria-label refers to something more specific: an attribute which specifies the text of an entire label (not just part of the label) for an element. Also, people might get confused and think this will replace the label for the button. Perhaps we could just call this "description" or "badgeDescription"; i.e. it describes the badge. If it's essential that we make it clear this is related to a11y, perhaps we could go with "a11yDescription".

Reply:
`badgeDescription` sounds good.

Reply:
> 
> 
> r+ with comments addressed. I tested and this works as expected; bravo and thank you. However, before you land, I'd like @MarcoZehe to take this for a spin to confirm he's happy with the UX now that he can actually try it out.
> 
> Marco, [here's a try build](https://treeherder.mozilla.org/#/jobs?repo=try&revision=1b165d7c25af2e7966f970f2eb6efe531c68faaf). If you test it with a new profile, the What's New button will appear on the toolbar automatically and it will be badged.

So I created a new profile, and the What's New button did appear, but it did not say anything other than "What's New button" when I arrowed to it via the keyboard. I also inspected it, and it didn't contain anything other than that label, no description. So there seems to be something missing that's in your local build, but not the try build.

Reply:
I've been able to reproduce the issue , not sure why we did not hit this earlier. I will update the PR and post a new try build link.

Reply:
Update: It looks like for fresh profiles there's a race condition between showing the notification and inserting the `asrouter.ftl` strings file. The badge is added first and Fluent doesn't know where to get the strings (that's why we don't get show `New feature:`).
It doesn't seem to reproduce for 2nd profile runs (and onward) so that's why we probably missed this.

Reply:
Here is the new try build for windows that attempts to fix the issue: https://treeherder.mozilla.org/#/jobs?repo=try&revision=efd6d3fd07394713578c304e5495fb8926d2356d